storage/raft_log_queue_test.go

```diff
@@ -57,10 +57,12 @@ func TestGetTruncatableIndexes(t *testing.T) {
 		t.Error(err)
 	}
 
+	r.RLock()
 	firstIndex, err := r.FirstIndex()
```

Comment:
When the call that needs the lock is not in the `if` statement, I'd rather release the lock before the `if err != nil` block (throughout this change).